addons/l10n_bf/data/account_tax_report_data.xml

```diff
@@ -420,6 +420,7 @@
                         <field name="engine">aggregation</field>
                         <field name="formula">BF_DEDUCTIBLE.balance + BF_CREDIT_REPORTED.balance - BF_CREDIT_ASKED.balance + BF_CANCELLED.balance + BF_UNPAID_CREDIT.balance + BF_OTHER_DEDUCTION.balance - BF_GROSS_TOTAL.balance</field>
                         <field name="subformula">if_above(XOF(0))</field>
+                        <field name="carryover_target" eval="False"/>
```

Comment:
I wonder how it works - given that the validation error is raised prior to writing anything. Setting `eval=False` works differently from a regular write? I thought the point was to execute SQL in order to work around the ORM.

Reply:
Regular write would try to find value in list values if not it would just take the values already present in database. Eval false would just unset value of carryover_target.

Reply:
and its raised prior to writing anything, I check with different data, its raised at the time of writing.

Reply:
Ahh indeed, now I remember it's been done at some point somewhere else already. I don't recall the place now tough. Anyways, if it gets a fw, I think it will be better than the script in `odoo/upgrade`.